skimage/morphology/convex_hull.py

```diff
@@ -128,8 +128,8 @@ def convex_hull_object(image, *, connectivity=2):
     objects may overlap in the result. If this is suspected, consider using
     convex_hull_image separately on each object or adjust ``connectivity``.
     """
-    if image.ndim > 2:
-        raise ValueError("Input must be a 2D image")
+    if image.ndim > 3:
+        raise ValueError("Input must be a 2D or 3D image")
```

Comment:
Why this change in this PR?

Reply:
my mistake– it shouldn't. fixing now

Reply:
Thank you!

Reply:
I thought @rubywerman was fixing #4865 at the same time :-) (but it can be in another PR, with a test ;-))